CONTRIBUTING.md

```diff
@@ -105,6 +106,14 @@ bazel test -c opt -k \
 //tensorflow_addons/...
 ```
 
+Note that the commands run with 4 GPUs with 8 tests per GPU by default.
+Please use the following variables to controll the hardware specifications:
+```
+export TF_GPU_COUNT=4 # Specify number of GPUs available
+export TF_TESTS_PER_GPU=8 # Specify number of tests per GPU
+export TF_PER_DEVICE_MEMORY_LIMIT_MB=1024 # Limit the memory used per test
+```
+
```

Comment:
Can you move this up so that it goes with the `run_docker.sh` way of running GPU tests. For the manual way that's right above this -- these shouldn't be required.

Reply:
Fixed

Reply:
Hmm, it might be wrong if we move it up because the env vars outside the container might not take effect inside the container.

Reply:
it turns out we have to do something like
```
bash tools/run_docker.sh -d gpu -c 'TF_GPU_COUNT=1 && make gpu-unit-test'
```
What to you think, Sean?